frontend/src/metabase/ui/components/inputs/DatePicker/DatePicker.styled.tsx

```diff
@@ -8,6 +8,7 @@ export const getDatePickerOverrides =
         /**
          * Months have different number of day rows (4, 5 or 6). This causes date picker height to change when
          * navigating between months, and the "next" & "previous" buttons will shift their positions (metabase#39487).
+         * This value should be the same as the default height of the calendar when 6 day rows are displayed.
          */
         mih: 314,
```

Comment:
Is there any way to set this value programmatically instead?
i.e. To take the height of a row and multiply it by 6. Of course, add whatever else's needed on top of that to reach 314.

Reply:
It's probably an overkill :thinking:

Reply:
@kamilmielnik should it go to Calendar? Unclear, just asking

Reply:
> @kamilmielnik should it go to Calendar? Unclear, just asking

Yeah, probably :+1: 
Done: cd0fc17f78bed9d837906bef144a03f4a29e6b25